packages/blocks/src/api/serializer.js

```diff
@@ -354,7 +365,16 @@ export function serializeBlock( block, { isInnerBlocks = false } = {} ) {
 	}
 
 	const blockType = getBlockType( blockName );
-	const saveAttributes = getCommentAttributes( blockType, block.attributes );
+	let saveAttributes = getCommentAttributes( blockType, block.attributes );
+
+	if ( __experimentalExcludeNonCopyableAttributes ) {
+		saveAttributes = omit(
+			saveAttributes,
+			__experimentalFilterBlockAttributes( blockName, {
+				__experimentalSupports: { copy: false },
```

Comment:
this here feels like it could be the thing exposed through `serializeBlock`'s options. do we need both the inner and outer language of talking about it?